README.md

```diff
@@ -1,2 +1,31 @@
-# fuelup
-The Fuel toolchain installer
+# fuelup: the Fuel toolchain installer
+
+`fuelup` installs the Fuel toolchain from our official release channels, enabling you to easily keep the toolchain updated.
+
+## Installation
+
```

Comment:
Maybe at the beginning of installation we should clarify which platforms we support (mostly for the sake of Windows native readers).